src/app/config/routes.tsx

```diff
@@ -2,11 +2,13 @@ import React, { ComponentClass } from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 import MainPage from '@core/pages/main';
+import MyDynamicRoute from '@core/pages/dynamic-route/dynamic-route';
```

Comment:
Use dynamic module here. to have whole component loaded dynamicly.
Currently you have pages/dynami-route in the same bundle and you are only loadidng dynamicly dynami-componenty.

the idea is to have whole route and all its containing component loaded dynamicly.

Reply:
fixed

Reply:
According to https://medium.com/javascript-in-plain-english/code-splitting-for-efficient-loading-how-to-use-react-lazy-imports-suspense-and-hoc-fabc8fb6525a Suspense should be in use, but please check if component is lazy initialized with the current solution

Reply:
resolved